docs/modules/ROOT/pages/general-info/general-info.adoc

```diff
@@ -1,138 +1,107 @@
 [[overview]]
 = Documentation Overview
 
-This page provides some general information about the Payara Project.
+This page provides some general information about the Payara Platform Enterprise Edition.
 
 [[release-strategy]]
 == Release Strategy
 
-Payara project has a public release every quarter. Each release incorporates fixes and enhancements from the Payara team, GlassFish upstream, and the community.
+The Payara Platform Community Edition project is released each month. Each release incorporates fixes and enhancements from the Payara team, GlassFish upstream, and Enterprise only features.
```

Comment:
Should be Enterprise edition instead

Reply:
Also: "Enterprise-only"

Reply:
Also, the Glassfish upstream cherry pick section was removed, so it might be better to remove mention of it here too.